mailpoet/lib/EmailEditor/Engine/Renderer/ContentRenderer/ContentRenderer.php

```diff
@@ -87,15 +87,55 @@ private function reset() {
    */
   private function inlineStyles($html, WP_Post $post) {
     $styles = (string)file_get_contents(dirname(__FILE__) . '/' . self::CONTENT_STYLES_FILE);
+
+    // Apply default contentWidth to constrained blocks.
+    $layout = $this->settingsController->getLayout();
+    $styles .= sprintf(
+      '
+      .is-layout-constrained > *:not(.alignleft):not(.alignright):not(.alignfull):not(.alignwide) {
```

Comment:
I added these rules so our defined contentSize in theme.json was applied by default. Without this, everything was full width during render, but narrow in editor.